docs/Makefile

```diff
@@ -175,3 +175,12 @@ pseudoxml:
 	$(SPHINXBUILD) -b pseudoxml $(ALLSPHINXOPTS) $(BUILDDIR)/pseudoxml
 	@echo
 	@echo "Build finished. The pseudo-XML files are in $(BUILDDIR)/pseudoxml."
+
+makedocsmessages: gettext
+	cd docs && sphinx-intl update -p _build/locale -l en
+
+ensurecrowdinclient:
+	ls -l crowdin-cli.jar || wget https://crowdin.com/downloads/crowdin-cli.jar # make sure we have the official crowdin cli client
+
+uploaddocsmessages: ensurecrowdinclient
+	java -jar crowdin-cli.jar upload sources -b `git symbolic-ref HEAD | xargs basename`
```

Comment:
This looks a bit dangerous, I mean just because it won't necessarily reflect the intention of the human or CI invoking `uploaddocsmessages`.

Wouldn't it be better to be able to hard-code specify which branch we wanna direct the upload to/from?